.github/workflows/pull_request_full_build.yml

```diff
@@ -120,6 +120,12 @@ jobs:
           done
         shell: bash
 
+#      - name: Checkout non-default branch
```

Comment:
We can remove this.

Reply:
Lets keep this commented since we frequently need this to be enabled